### ISSUE TITLE
skip slides using links

### ISSUE THREAD

**Issue by neotheicebird** [jdan/cleaver#101]
I need to go from one slide to another using a hyperlink.

I am in basic.html#3 and I want to go to basic.html#7, I added a hyperlink to a text with link './basic.html#7'. On clicking it the address in the address bar changes, but the slide doesn't change. Is there any way to achieve this?

**Comment by jdan** [comment]
Good idea :) Right now you'll need to inject your own JS and use the `navigate` function which is exposed to the client.

The front-end JS needs a big overhaul, though, so I'll be sure to include something like this.

**Comment by neotheicebird** [comment]
Cool, thanks. Let me try adding the JS for `navigate` in my markdown and see if it works.

**Comment by tsycho** [comment]
+1 for this feature.

**Comment by tsycho** [comment]
@neotheicebird Were you able to do this? Do you have a code example of how you used the navigate function?
